lib/rucio/web/rest/flaskapi/v1/rules.py

```diff
@@ -307,13 +298,11 @@ def post(self, rule_id):
         parameters = json_parameters()
         rse_expression = param_get(parameters, 'rse_expression')
         rule_id = param_get(parameters, 'rule_id', default=rule_id)
-        activity = param_get(parameters, 'activity', default=None)
-        source_replica_expression = param_get(parameters, 'source_replica_expression', default=None)
+        override = param_get(parameters, 'override', default=None)
```

Comment:
The client can only change the `activity` and `source_replica_expression`. The rest api can override all other field too. Is that a problem?

Reply:
I think this is good. Advanced users might want additional control (e.g. changing the `comment`), which may at some point come to the CLI.